src/main/resources/l10n/JabRef_en.properties

```diff
@@ -1172,7 +1172,7 @@ The_chosen_date_format_for_new_entries_is_not_valid=The_chosen_date_format_for_n
 
 The_chosen_encoding_'%0'_could_not_encode_the_following_characters\:=The_chosen_encoding_'%0'_could_not_encode_the_following_characters\:
 
-The_entry_is_the_member_of_groups=The entry is the member of groups
+The_entry_is_the_member_of_groups=The entry is the member of groups: 
```

Comment:
The values on the left and on the right must be equal in the English properties file